### DISCUSSION TITLE
HELP!! Security Onion web interface giving me gray hairs!!!!! Spent the last 2 weeks trying to install Security Onion. Can't access web interface

### DISCUSSION THREAD

**Discussion by ProtonmailLA** [Security-Onion-Solutions/securityonion#13173] [2.4]
### Version

2.4.30

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Eval

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

8

### RAM

14

### Storage for /

100

### Storage for /nsm

100

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

I can't access Security Onion web interface on 192.168.12.130 from a browser, which is what I set as the IP of the management network of the Security Onion. I can ping hosts on other networks from the terminal of the security onion though. I'm also able to ping 192.168.12.130 from outside the network too. Using 2 NICs. 1st is NAT, second is bridged. IP settings for Security Onion are all correct. ufw firewall is enabled, and web interface is allowed on both port 80 and 443. All containers are running. Still can't access web interface.

Done this: so-firewall includehost analyst 192.168.12.130

![Oracle Linux 9 64-bit-2024-06-09-18-48-27](https://github.com/Security-Onion-Solutions/securityonion/assets/98847384/a918c7f3-31a8-4dea-9599-2930bf1a4b35)

Also, I'm using Oracle Linux 9 (64bit). Don't know if that makes any difference.

I don't understand this. Why can't I access the web interface? 

(UPDATE) Did a fresh (airgap) install on Ubuntu 64bit version. NAT for management and Bridged for monitoring. Installation was successful, but I STILL CAN'T ACCESS WEB INTERFACE ON 192.168.12.131. All the containers are running fine.
(Am I supposed to be able to ping a host on another network on Airgap though)

HELP!!!!! I've been running this for weeks!!!!!!!!!!!!!

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment]
Please do not create duplicate discussions:
https://github.com/Security-Onion-Solutions/securityonion/discussions/13088
